docs/docs/hpc-on-k8s/hadoop.md

```diff
@@ -0,0 +1,56 @@
+# Hadoop Workload
+Hadoop is an open-source framework that efficiently processes and stores vast amounts of data across distributed computing clusters. It breaks down large datasets into manageable chunks and distributes them across multiple computers, allowing for parallel processing and analysis. This distributed approach makes it possible to handle massive amounts of data that would be impractical to process on a single machine.
+
+
+## Steps to Execute Job
+Below is the example manifest file for Hadoop Mapreduce workload in a Kubernetes cluster.
+```yaml
+# job.yaml
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: hadoop-
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  parallelism: 1
+  completions: 1
+  suspend: true
+  template:
+    spec:
+      containers:
+      - name: dummy-job
+        image: shubhamdang/my_hadoop_image:latest
+        command: [ "python3", "/opt/main.py" ]
+        env:
+          - name: MINIO_IP
+            value: "10.151.15.78"
+          - name: MINIO_PORT
+            value: "31252"
+          - name: MINIO_AKEY
+            value: "Mq6wmeNk0NOc0vD9Efut"
+          - name: MINIO_SKEY
+            value: "Z3ETBqC3GuIiU9PomjBbmmC5h8I5I7WgN1wNWlCG"
```

Comment:
Not sure if these are sensitive secret keys or example ones but may be worth noting in the documentation what these keys should be set to (and maybe where to find them?)

Reply:
Added doc in minio to create access key added the the link of minio in workload notes.